flink/v1.19/flink/src/test/java/org/apache/iceberg/flink/source/TestStreamScanSql.java

```diff
@@ -48,7 +48,9 @@
 import org.junit.jupiter.api.AfterEach;
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.TestTemplate;
+import org.junit.jupiter.api.Timeout;
 
+@Timeout(60)
```

Comment:
is this needed?

Reply:
Yeah. My first change (where the validation always happened) caused the test streaming jobs to stuck in restart loop. So the test never finished, and never failed. Took me some time to find the offending test 😢